File: book-service/src/main/java/br/com/wellingtoncosta/bookservice/domain/model/Book.java
package br.com.wellingtoncosta.bookservice.domain.model;

import lombok.Data;

import java.time.LocalDate;

/**
 * @author Wellington Costa on 30/11/2018.
 */
@Data public class Book {

    private Long id;
    private String tiile;
    private String author;
    private String description;
    private LocalDate releaseDate;

}

File: book-service/src/main/kotlin/br/com/wellingtoncosta/book/Book.kt
package br.com.wellingtoncosta.book

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

/**
 * @author Wellington Costa on 24/07/18.
 */
@Document
data class Book(
        @Id var id: String,
        var title: String,
        var description: String = "",
        var author: String,
        var year: String
)